src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -0,0 +1,907 @@
+                    AssertNullableAttribute(method.GetReturnTypeAttributes());
+                });
+        }
+
+        [Fact]
+        public void EmitAttribute_LocalFunctionParameters()
+        {
+            var source =
+@"class C
+{
+    static void M()
+    {
+        void L(object? x, object y) { }
+        L(null, 2);
+    }
+}";
+            CompileAndVerify(
+                source,
+                parseOptions: TestOptions.Regular8,
+                options: TestOptions.DebugDll.WithMetadataImportOptions(MetadataImportOptions.All),
+                symbolValidator: module =>
+                {
+                    var method = module.ContainingAssembly.GetTypeByMetadataName("C").GetMethod("<M>g__L|0_0");
+                    AssertNullableAttribute(method.Parameters[0].GetAttributes());
+                    AssertNoNullableAttribute(method.Parameters[1].GetAttributes());
+                });
+        }
+
+        [Fact]
+        public void EmitAttribute_ExplicitImplementationForwardingMethod()
+        {
+            var source0 =
+@"public class A
+{
+    public object? F() => null;
+}";
+            var comp0 = CreateStandardCompilation(source0, parseOptions: TestOptions.Regular8);
+            var ref0 = comp0.EmitToImageReference();
+            var source =
+@"interface I
+{
+    object? F();
+}
+class B : A, I
+{
+}";
+            CompileAndVerify(
+                source,
+                additionalRefs: new[] { ref0 },
+                parseOptions: TestOptions.Regular8,
+                options: TestOptions.DebugDll.WithMetadataImportOptions(MetadataImportOptions.All),
+                symbolValidator: module =>
+                {
+                    var method = module.ContainingAssembly.GetTypeByMetadataName("B").GetMethod("I.F");
+                    AssertNullableAttribute(method.GetReturnTypeAttributes());
+                });
+        }
+
+        [Fact]
+        public void EmitAttribute_Iterator()
+        {
+            var source =
+@"using System.Collections.Generic;
+class C
+{
+    static IEnumerable<object?> F()
+    {
+        yield break;
+    }
+}";
+            CompileAndVerify(
+                source,
+                parseOptions: TestOptions.Regular8,
+                options: TestOptions.DebugDll.WithMetadataImportOptions(MetadataImportOptions.All),
+                symbolValidator: module =>
+                {
+                    var property = module.ContainingAssembly.GetTypeByMetadataName("C").GetTypeMember("<F>d__0").GetProperty("System.Collections.Generic.IEnumerator<System.Object>.Current");
+                    // Generated iterator methods have no [Nullable] attributes.
+                    // Since the methods are only called through IEnumerable<T>
+                    // or IEnumerator<T>, that is not an issue.
+                    AssertNoNullableAttribute(property.GetAttributes());
```

Comment:
>AssertNoNullableAttribute(property.GetAttributes()); [](start = 20, length = 52)

Please test attributes on the accessor's return type as well.

Reply:
Added.